invenio_vocabularies/services/schema.py

```diff
@@ -24,15 +24,15 @@ class BaseVocabularySchema(BaseRecordSchema):
     """Base schema for vocabularies."""
 
     title = i18n_strings
-    description = i18n_strings
-    icon = fields.Str(allow_none=False)
 
 
 class VocabularySchema(BaseVocabularySchema):
     """Service schema for vocabulary records.."""
 
-    tags = fields.List(SanitizedUnicode())
-    type = fields.Str(attribute='type.id', required=True)
+    description = i18n_strings
```

Comment:
They cannot be left above? Even id they are not used in the affiliations vocab. From the specs we wrote, we chose this as common props.

Reply:
You mean to *accept them if they exist*? I thought we wanted to enforce a more restricted schema. I can put them back, but will they always be empty? It would affect also jsonschema and ES mappings.